### ISSUE TITLE
double objectives on tooltips

### ISSUE THREAD

**Issue by x003** [AeroScripts/QuestieDev#228]
still experiencing this in the latest revision
![bug](https://cloud.githubusercontent.com/assets/12950645/14939257/0879ebb6-0f0d-11e6-9fc7-b1aa6b30abb8.png)

**Comment by Dyaxler** [comment]
Turn off the tooltip settings in EQL3...

![capture](https://cloud.githubusercontent.com/assets/17400948/14939373/a12523ae-0ef8-11e6-8454-f7dbd344d107.JPG)

**Comment by x003** [comment]
Maybe this is a shagu issue again? :/

![untitled](https://cloud.githubusercontent.com/assets/12950645/14941889/3021adbc-0f7a-11e6-8c54-38659d5f56fb.png)

**Comment by Dyaxler** [comment]
You're running both Shagu and Questie? That's not supported. Shagu does insert things into the tooltips. What I had you turn off above is an Extended QuestLog option not Shagu. But EQL3 does come with Shagu and it's been modified from the original so I can't guarantee compatibility.

**Comment by x003** [comment]
Okay I switched to the supported EQL3 3.6.1 and I still get this issue

**Comment by Dyaxler** [comment]
Can you send me a copy of your !Questie.lua and AddOns.txt files from your WTF directory?

**Comment by x003** [comment]
After some testing I figured out that this only occurs when I have the TheoryCraft addon enabled for some reason

**Comment by Dyaxler** [comment]
That kind of makes sense... TheoryCraft does it's own tool-tip injection. I've never run it and have no idea how they implemented it. I did track down a bug in Questie's tooltip code and it's related to quest items specifically found outside of a mobs drop table or a mob itself as a quest objective. Items contained in something. Not sure if it's a unique quest item or not as I was only able to re-pro the issue once but I did confirm a specific block of code that it's looping through twice. I've pointed it out to the original contributors and am getting their feed back on it before I attempt to write a fix. This block is in a function that has several situational checks in it and I'm not 100% sure what they all do and I'm afraid that I might break or introduce a new issue so for now I'm leaving it alone.

**Comment by Muehe** [comment]
@x003 Since Eric has discontinued development of ShaguDB, I have continued my [WHDB](https://github.com/Muehe/WHDB) fork (the addon ShaguDB was forked from as well). It doesn't have all the features of ShaguDB (or Questie for that matter) but it should be there soon™.

**Comment by Dyaxler** [comment]
This will be fixed in version 3.1 which I'm preparing to release now.

**Comment by iwanttoeatyo** [comment]
Im still getting the duplicated quest drops while running TheoryCraft v1.06.5 and Questie v3.1 together.
Quick fix is to edit TheoryCraftTooltip.lua in the TheoryCraft addon and remove line 46.
I just changed
```
end
frame:Show()
return frame
```
 to 
```
end
--[[frame:Show()]]--
return frame
```
All the tooltips seem to still be working properly for me.
